packages/core/src/Table/stories/TableSection/TableEditable.tsx

```diff
@@ -0,0 +1,602 @@
+                    <HvInput
+                      className={classes.inputRoot}
+                      defaultValue={
+                        edit ? row.original[String(col.accessor)] : undefined
+                      }
+                      placeholder={col.Header}
+                      onChange={(event, value) => {
+                        if (edit) {
+                          setEditRows((prev) =>
+                            prev.map((r) =>
+                              r.row.id === row.original.id
+                                ? {
+                                    dirty: true,
+                                    row: {
+                                      ...r.row,
+                                      [String(col.accessor)]: value,
+                                    },
+                                  }
+                                : r
+                            )
+                          );
+                          return;
+                        }
+                        setNewRow((prev) => ({
+                          ...prev,
+                          [String(col.accessor)]: value,
+                        }));
+                      }}
+                    />
+                    {last && (
+                      <HvButton
+                        icon
+                        aria-label="Delete"
+                        variant="secondaryGhost"
+                        onClick={
+                          edit
+                            ? () => {
+                                handleDelete(row);
+                                setEditRows((prev) =>
+                                  prev.filter(
+                                    (r) => r.row.id !== row.original.id
+                                  )
+                                );
+                              }
+                            : handleCancelAddRow
+                        }
+                      >
+                        <Delete />
+                      </HvButton>
+                    )}
+                  </div>
+                </div>
+              </HvTableCell>
+            );
+          })}
+        </HvTableRow>
+        <HvTableRow
+          key={`editable_row_actions_${row?.original.id}`}
+          role="row"
+          className={classes.tableRowEditable}
+        >
+          <HvTableCell
+            colSpan={columns.length}
+            key={`editable_row_cell_actions_${row?.original.id}`}
+            role="cell"
+            className={classes.tableCellEditable}
+          >
+            <div className={edit ? undefined : classes.slide}>
+              <div className={classes.editableActions}>
+                <HvButton
+                  variant="primaryGhost"
+                  onClick={
+                    edit ? () => handleEditRow(row.original.id) : handleAddRow
+                  }
+                >
+                  Save
+                </HvButton>
```

Comment:
We should use a `form` with a submit button here

Reply:
Yes, I'll add the `form`